rest/api.go

```diff
@@ -153,27 +153,59 @@ func (h *handler) handleFlush() error {
 
 }
 
-func (h *handler) handleResync() error {
-
-	//If the DB is already re syncing, return error to user
+func (h *handler) handleGetResync() error {
 	dbState := atomic.LoadUint32(&h.db.State)
 	if dbState == db.DBResyncing {
-		return base.HTTPErrorf(http.StatusServiceUnavailable, "Database _resync is already in progress")
+		h.writeRawJSON([]byte(`{"status": "running"}`))
```

Comment:
Reporting a 'document processed' count here would be a really nice feature - any ideas on how much work that would entail?

Reply:
Added ResyncStatus in DatabaseContext to handle